rele/middleware.py

```diff
@@ -15,7 +15,7 @@ def register_middleware(config):
         module_path = ".".join(module_parts)
         module = importlib.import_module(module_path)
         middleware = getattr(module, middleware_class)()
-        middleware.setup(config)
```

Comment:
Seeing this again, we probably could do without `middleware.setup`. All that could be done in the init